src/main/xerus/monstercat/api/Player.kt

```diff
@@ -167,31 +167,23 @@ object Player: FadingHBox(true, targetHeight = 25) {
 		}
 	}
 	
-	private val pauseButton = ToggleButton().id("play-pause")
-			.onClick { if (isSelected) player?.pause() else player?.play() }
-			.apply { tooltip = Tooltip("Pause / Play") }
+	private val pauseButton = ToggleButton().id("play-pause").onClick {
+		if (isSelected) player?.pause() else player?.play()
+	}.apply { tooltip = Tooltip("Pause / Play") }
```

Comment:
The previous version looked better and is what I'd like to see for all of these

Reply:
New version 7660b501f1ce1fa0c1038089fc246e23af288433